dm_control/utils/inverse_kinematics_test.py

```diff
@@ -90,7 +139,7 @@ def testQposFromSitePose(self, target, inplace):
     while True:
       result = ik.qpos_from_site_pose(
           physics=physics2,
-          site_name=_SITE_NAME,
+          sites_names=[_SITE_NAME],
```

Comment:
The tests should prove that the existing functions continue to work.

Maybe add a parameterized annotation that will run both qpos_from_site_pose and qpos_from_site_poses on the same input.